src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Logging/ISuppressionEngine.cs

```diff
@@ -1,5 +1,5 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Why are making this change? dotnet/runtime uses the existing license header everywhere: https://github.com/dotnet/runtime/blob/5d370f31305012b074f070337ace39ca1b5aedeb/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs#L1-L2

cc @jkotas

Reply:
"Licensed to the .NET Foundation under one or more agreements." is the correct header to use in .NET Foundation projects.

See https://github.com/dotnet-foundation/foundation/blob/master/guidance/copyright.md#file-headers . 

dotnet/runtime further shortened it to 2 lines, both the make it shorter and to address some problems with the 3 line version. See https://github.com/dotnet/runtime/pull/38732 for rationale.

Reply:
Sounds good. I'll update the editorconfig based on dotnet/runtime repo and run the quick fix again.